views/js/qtiCreator/editor/mathInput/mathInput.js

```diff
@@ -55,30 +55,98 @@ define([
             log:        { label: 'log',         latex: '\\log',         fn: 'cmd',      desc: __('Log') },
             ln:         { label: 'ln',          latex: '\\ln',          fn: 'cmd',      desc: __('Ln') },
+
+            // Constants and brackets
             e:          { label: '&#8494;',     latex: '\\mathrm{e}',   fn: 'write',    desc: __('Euler\'s constant') },
             infinity:   { label: '&#8734;',     latex: '\\infty',       fn: 'cmd',      desc: __('Infinity') },
+            colon:      { label: ':',           latex: ':',             fn: 'write',    desc: __('Colon') },
             lbrack:     { label: '[',           latex: '\\lbrack',      fn: 'cmd',      desc: __('Left bracket') },
             rbrack:     { label: ']',           latex: '\\rbrack',      fn: 'cmd',      desc: __('Right bracket') },
-            pi:         { label: '&pi;',        latex: '\\pi',          fn: 'cmd',      desc: __('Pi') },
-            cos:        { label: 'cos',         latex: '\\cos',         fn: 'cmd',      desc: __('Cosinus') },
-            sin:        { label: 'sin',         latex: '\\sin',         fn: 'cmd',      desc: __('Sinus') },
-            lte:        { label: '&le;',        latex: '\\le',          fn: 'cmd',      desc: __('Lower than or equal') },
+            lparen:     { label: '(',           latex: '(',             fn: 'write',    desc: __('Left parenthesis') },
+            rparen:     { label: ')',           latex: ')',             fn: 'write',    desc: __('Right parenthesis') },
+            lte:        { label: '&le;',        latex: '\\le',          fn: 'cmd',      desc: __('Less than or equal') },
             gte:        { label: '&ge;',        latex: '\\ge',          fn: 'cmd',      desc: __('Greater than or equal') },
+
+            // Trigonometry and functions
+            degrees:    { label: 'x°',          latex: '\^\\circ',      fn: 'write',    desc: __('Degrees') },
+            pi:         { label: '&pi;',        latex: '\\pi',          fn: 'cmd',      desc: __('Pi') },
+            sin:        { label: 'sin',         latex: '\\sin',         fn: 'cmd',      desc: __('Sine') },
+            cos:        { label: 'cos',         latex: '\\cos',         fn: 'cmd',      desc: __('Cosine') },
+
+            // Operators
             times:      { label: '&times;',     latex: '\\times',       fn: 'cmd',      desc: __('Multiply') },
             divide:     { label: '&divide;',    latex: '\\div',         fn: 'cmd',      desc: __('Divide') },
-            plusminus:  { label: '&#177;',      latex: '\\pm',          fn: 'cmd',      desc:__( 'Plus/minus') }
+            plusminus:  { label: '&#177;',      latex: '\\pm',          fn: 'cmd',      desc: __('Plus/minus') },
+            equals:     { label: '=',           latex: '=',             fn: 'write',    desc: __('Equals') },
+
+            // Comparison
+            equiv:      { label: '≡',           latex: '\\equiv',       fn: 'cmd',      desc: __('Equivalent') },
```

Comment:
Isn't it the wrong symbol? The upper line should be `~`: https://www.compart.com/en/unicode/U+2245 and the wording should be `Congruent`.